Dungeoneer/src/com/interrupt/dungeoneer/dto/LookAtDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.interrupt.dungeoneer.dto;
+
+import com.badlogic.gdx.graphics.Color;
+
+public class LookAtDTO {
+    private String title;
+    private String attributes;
+    private Color color;
+
+    public LookAtDTO(String title, String attributes, Color color) {
+        this.title = title;
+        this.attributes = attributes;
```

Comment:
I wasn't super sure what attributes meant here. Maybe call this additional info?